CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 
 ## Unreleased
 
+#### Fixed
+
+* mock.Protected().Setup<int>("Foo") fails if base implementation of the method is hidden in the derived class (#1341)
+
```

Comment:
The changelog is currently out of sync with the latest releases. (Whats's currently titled "Unreleased" became the last 4.18.* release, and the 4.20.* releases are completely missing.) It should first be updated (in a separate PR) so that your changelog entry will end up with the correct release instead of with the 4.18.* changes that were already published.

Reply:
Should be up to date now.